lib/call-session.js

```diff
@@ -521,6 +521,13 @@ class CallSession extends Emitter {
               '-Allow',
               '-Session-Expires'
             ],
+            // Add X-CID header to feature server response.
+            // to allow calling/status hooks contains sip_callid.
+            responseHeaders: (uacRes, headers) => {
+              if (this.req.locals.cdr) {
+                headers['X-CID'] = this.req.locals.cdr.sip_callid;
```

Comment:
I prefer to reference this everywhere as sbc_callid

Reply:
fixed it in feature server